packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/process/dr-response.ts

```diff
@@ -44,10 +44,10 @@ export type DocumentResponse = {
 };
 
 let s3UtilsInstance = new S3Utils(region);
-export function getS3UtilsInstance(): S3Utils {
+function getS3UtilsInstance(): S3Utils {
   return s3UtilsInstance;
 }
-export function setS3UtilsInstance(s3Utils: S3Utils): void {
+export function setS3UtilsInstanceForStoringDrResponse(s3Utils: S3Utils): void {
```

Comment:
Can we keep the original name? This is a "set" function for the respective "get" function and variable, so it's helpful if the names match.

If the client (code that imports it) wants to name it differently, it can do so like this:

```ts
import { setS3UtilsInstance as setS3UtilsInstanceForStoringDrResponse } from "...";
```